src/encoded/audit/experiment.py

```diff
@@ -133,12 +141,37 @@ def audit_experiment_chipseq_control_read_depth(value, system, files_structure):
                 target_investigated_as)
 
 
-def get_control_target_name(control_id, control_objects):
+def check_control_target(control_id, control_objects, bam_id, bam_type):
```

Comment:
This isn't a big deal, but I wouldn't change functions name with out good reason.  In this case get_control_target_name only exists in this file but the code reviewer needs to check that.  Also, based on the naming conventions a prefix of 'check' leads me to think the function is boolean.  

Just something to consider when updating code.  A better name might be get_control_target_failures?